template.tpl

```diff
@@ -1112,6 +1112,29 @@ ___TEMPLATE_PARAMETERS___
           }
         ],
         "valueHint": "/jrf/renew"
+      },
+      {
+        "type": "TEXT",
+        "name": "cookie_keeper_host",
+        "displayName": "Cookie Keeper host",
+        "simpleValueType": true,
+        "canBeEmptyString": true,
+        "help": "Cloudflare's Cookie Keeper Worker Subdmain (optional)",
+        "enablingConditions": [
+          {
+            "paramName": "tag_type",
+            "paramValue": "init",
+            "type": "EQUALS"
+          }
+        ],
+        "valueHint": "https://ck.journify.io"
+      },
+      {
+        "type": "TEXT",
+        "name": "user_agent",
+        "displayName": "User Agent",
+        "simpleValueType": true,
+        "defaultValue": "{{journify_user_agent}}"
```

Comment:
Adding a default value of type `variable` doesn't seem like a good default. I would suggest keeping it empty and add a help text.